Dockerfile.ui-monitoring

```diff
@@ -6,15 +6,14 @@ USER 0
 
 ENV HUSKY=0
 COPY ui-monitoring/web/package*.json web/
-COPY ui-monitoring/Makefile Makefile
-RUN make install-frontend-ci
+RUN cd web && npm ci --omit=optional --ignore-scripts
```

Comment:
any reason we cannot use the make commands?

Reply:
seems like cachi2 is having issues with makefiles and prefetching dependencies, and having that around makes npm ignore the precached deps and trying to download only to fail in the disconnected environment. I'm trying this workaround for now.